### ISSUE TITLE
Colab Error

### ISSUE THREAD

**Issue by krummrey** [omidsakhi/lorakit#1]
I was trying to run it on Colab and got this error:
```
Job config/train_lora_sdxl_myLORA.yaml failed with error: Repo id must be in the form 'repo_name' or 'namespace/repo_name': '/content/lorakit/models/sd_xl_base_1.0.safetensors'. Use `repo_type` argument if needed.
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/lorakit/cli/lorakit.py", line 31, in main
    job.run()
  File "/usr/local/lib/python3.10/dist-packages/lorakit/train.py", line 632, in run
    noise_scheduler, is_edm_scheduler = self._get_noise_scheduler()
  File "/usr/local/lib/python3.10/dist-packages/lorakit/train.py", line 475, in _get_noise_scheduler
    model_index = hf_hub_download(
  File "/usr/local/lib/python3.10/dist-packages/huggingface_hub/utils/_deprecation.py", line 101, in inner_f
    return f(*args, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/huggingface_hub/utils/_validators.py", line 106, in _inner_fn
    validate_repo_id(arg_value)
  File "/usr/local/lib/python3.10/dist-packages/huggingface_hub/utils/_validators.py", line 154, in validate_repo_id
    raise HFValidationError(
huggingface_hub.errors.HFValidationError: Repo id must be in the form 'repo_name' or 'namespace/repo_name': '/content/lorakit/models/sd_xl_base_1_0.safetensors'. Use `repo_type` argument if needed.

```
in the yaml I've defined:
```
  model:
    name_or_path: "/content/lorakit/models/sd_xl_base_1.0.safetensors"
    variant: "fp16"
```
The file has been loaded from hugginface and is present at the location.

And as the memory limit is probably the next roadblock, how would I get the requirement down to 12-16GB?
Lower the number of `num_workers` to 2 or 3?

**Comment by omidsakhi** [comment]
Thanks @krummrey for reporting this. I noticed that I forgot to remove the hard-coded model repo from the config. The name_or_path is the huggingface repo name or the location of the snapshot. It is a folder path. In Colab, if the model is missing, it will be downloaded automatically. 

As for the second issue, I added another config for T4 which enables gradient checkpointing and disables sampling to save memory and avoid loading another pipeline for image generation. The training is a bit slow though.

Changing the num_workers will not have any significant effect since they are only used for loading the images.
